tests/unit/test_parameters.py

```diff
@@ -9,15 +9,68 @@
 from databricks.sql.thrift_api.TCLIService.ttypes import (
     TSparkParameter,
     TSparkParameterValue,
+    TSessionHandle,
+    TOpenSessionResp,
 )
 from databricks.sql.utils import DbSqlParameter, DbSqlType
 import pytest
 
-from decimal import Decimal
+from databricks.sql.thrift_backend import ThriftBackend
+from databricks.sql.thrift_api.TCLIService import ttypes
 
+from decimal import Decimal
+from databricks.sql.client import Connection
 from typing import List
 
 
+class TestSessionHandleChecks(object):
+    def test_get_session_handle(self):
+        assert (
```

Comment:
For new tests we should have one assertion per test. Please refactor this either into separate tests or a single test using `pytest.mark.parametrize`.